tests/installation/boot_into_snapshot.pm

```diff
@@ -0,0 +1,54 @@
+# SUSE's openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2012-2016 SUSE LLC
```

Comment:
@aaannz IIRC it wasn't me :-) I think it should be as you said if starting from blank but only in this file as the other new files are basically "rename and adapt"